src/models/db/account.js

```diff
@@ -10,6 +11,24 @@ const EntryGroup = require('./entry-group').EntryGroup
 const Sequelize = require('sequelize')
 const sequelize = require('../../services/db')
 
+function hashPassword (password) {
+  // cache hashes of passwords used in tests to speed up tests 4X
```

Comment:
I'm all for speeding up tests, but it seems weird to have a hardcoded list of test passwords in this method. Can we pull the list from a file under `/test/data`?